app/client/src/features/events/Moderation/CreateModerator.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { graphql, useMutation } from 'react-relay';
-
+import { useSnack } from '@local/features/core';
```

Comment:
Nitpicky: This should be in the second block of imports.  External imports in first block, local ones in second block.